### PR TITLE
MD-69: Condition checking

### PR DESCRIPTION
**Pull request by alexandre-merle**
Check conditions for principal evaluation

Fixes https://scality.atlassian.net/browse/MD-69

### CODE REVIEW

**Comment by alexandre-merle** [comment]
@ironman-machine try

File: lib/policyEvaluator/evaluator.js [hunk 1]
```diff
@@ -269,4 +269,6 @@ evaluators.evaluateAllPolicies = (requestContext, allPolicies, log) => {
     return verdict;
 };
 
+evaluators.meetConditions = meetConditions;
```
Comment (philipyoo) [inline lib/policyEvaluator/evaluator.js]: Just a small suggestion, but maybe keep the same format as used for other methods like `evaluateAllPolicies`?

**Comment by alexandre-merle** [comment]
@philipyoo updated and rebased :)

**Comment by rahulreddy** [comment]
Thanks @philipyoo !

**Comment by rahulreddy** [comment]
@ironman-machine r+

**Pull request merged**